connector/connect/common/src/test/resources/query-tests/explain-results/function_shiftleft.explain

```diff
@@ -1,2 +1,2 @@
-Project [shiftleft(cast(b#0 as int), 2) AS shiftleft(b, 2)#0]
+Project [(cast(b#0 as int) << 2) AS (b << 2)#0]
```

Comment:
can we keep the pretty string unchanged? if people explicitly use `shiftleft` function, then the generated alias name should still be `shiftleft`.

Reply:
@yaooqinn can we fix this? I think it's wrong to use `<<` in the pretty string if users call `shiftleft`

Reply:
I will check this soon

Reply:
One more question here, shall we revert or keep changes in auto-generated shift expressions? Plans that are represented with symbols appear to have good readability.

Reply:
One more question here, shall we revert or keep changes in auto-generated shift expressions? Plans that are represented with symbols appear to have better readability than lowercased function names

Reply:
if people use the new shift syntax, then the pretty string should match it. Otherwise, we should keep the previous behavior.